webviz_config/_docs/_build_docs.py

```diff
@@ -162,3 +162,23 @@ def build_docs(build_directory: pathlib.Path) -> None:
     (build_directory / "sidebar.md").write_text(
         template.render({"packages": plugin_documentation.keys()})
     )
+
+
+def _split_docstring(docstring: str) -> List[str]:
+    """Divide docstring by splitting on ---, also unindent
+    in case of indented docstrings (similar to this one)
+    """
+    lines = docstring.strip().split("\n")
+    if len(lines) == 1:
+        return lines
+
+    indent_spaces = 1000  # far beyond max width
+    for line in lines[1:]:
+        if line.strip() == "":
+            continue
+        indent_spaces = min(indent_spaces, (len(line) - len(line.lstrip())))
```

Comment:
```suggestion
indent_spaces = min([len(line) - len(line.lstrip()) for line in lines[1:] if line.strip() != ""])
```
Not tested, but something along these lines might be possible + easier to read?